packages/react-admin-core/src/Tabs.tsx

```diff
@@ -31,6 +31,7 @@ interface IProps {
     appBarComponent?: React.ComponentType<AppBarProps>;
     tabComponent?: React.ComponentType<TabProps>;
     defaultIndex?: number;
+    onChange?: (value: number) => void;
```

Comment:
der onChange-Event von MuiTabs ist mit "handleChange" ja selbst in der Componente implementiert. 
Deine onChange-Funktion wird **nach** dem implementiereten change-handler  (`setValue(newValue)`) ausgeführt. Von daher find ich als Prop-Namen **onChanged** oder **afterChange** , besser. **onChange** suggeriert das der default change-handler ersetzt wird, was aber hier nicht der Fall ist.

Reply:
:+1: